mods/gen5/formats-data.js

```diff
@@ -3130,4 +3134,94 @@ exports.BattleFormatsData = {
 		eventOnly: true,
 		requiredItem: "Shock Drive",
 	},
+	ufo: {
+		inherit: true,
+		isNonStandard: false,
```

Comment:
Actually, wouldn't they still need `isNonStandard` to be false in order to be theoretically usable in a Gen V BH setting? Otherwise, if nothing is different from the main file, does this file even need changing?